### ISSUE TITLE
Duplicate "Edit" links on Satellite table in CMS Vet Centers Dashboard

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10296]
## Description
On the Vet Center Dashboard in the CMS screen, there is a table displaying all Satellite locations.  Within this table, each row has a call to action that says "Edit". This can cause duplicate links on the same page when there are multiple locations.  Ideally each link should have distinct, unique text to indicate to the user where the link would direct them - "Edit Lowell Vet Center - Danvers"

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/184973723-cfc53c86-1727-4b6e-8c95-be14d82c5e30.png)

## Accessibility Standard

WCAG version 2.0 AAA, [Criterion 2.4.9](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)

## Acceptance Criteria
- [ ] UX review to determine content updates for links
- [ ] Technical review
- [ ] Change management consulted
- [ ] Implementation ticket created

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by omahane** [comment]
I not that this same issue is true of the main Content page (`/admin/content`) and any view like it. In this regard, the scope of this should probably be all of CMS.

**Comment by jilladams** [comment]
@laflannery based on discussion in planning today and Christian's note above, I moved this ticket to the CMS team backlog, since it applies to all content views.

**Comment by FranECross** [comment]
@Agile6MSkinner I'm not sure which epic this ticket belongs to, but don't think it's the current "Outstations" epic.
